apps/client/src/components/divider/divider.tsx

```diff
@@ -1,4 +1,8 @@
+import React, { CSSProperties } from 'react';
 import styles from './divider.module.css';
-export const Divider: React.FC = () => {
-  return <div className={styles.divider} />;
+type Props = {
+  style: CSSProperties;
```

Comment:
Instead of passing a style I think a `className` would be better.
Unless we are using Tailwind it's better to split the css and tsx files as much as possible intot seperate files